Bugfix masthead and columnists

alicebartlett opened the pull request:
https://github.com/Financial-Times/o-ft-icons/issues/96

This PR addresses two problems, but because of the way the icon font is built (ie, I can't build anything on my machine unless I fix both problems) both of these problems are fixed in the same PR.

**Problem 1**
The section_columnists svg had an incomplete path. This incompleteness was causing the woff generated in the conversion to the icon font to display oddly on all the versions of Windows (8, 9, 10) that I could test on.

Before:
![screen shot 2015-11-30 at 14 50 55](https://cloud.githubusercontent.com/assets/68009/11475074/5c9f5b20-9773-11e5-8959-8e1f6e9a3b4e.png)

After:
![screen shot 2015-11-30 at 14 52 18](https://cloud.githubusercontent.com/assets/68009/11475086/665df9a0-9773-11e5-8cd0-46dd154aefcc.png)



**Problem 2**
The Financial Times masthead in the woff  is rendering with no negative space inside the A's on all Windows versions.
The fix to this appears to have been to re-order the order that the A and it's negative space were defined, so that the space is defined first

Before: 
![screen shot 2015-11-30 at 14 50 46]
(https://cloud.githubusercontent.com/assets/68009/11475158/c4bd578e-9773-11e5-99c7-315f8cceb826.png)

After: 
![screen shot 2015-11-30 at 14 52 25](https://cloud.githubusercontent.com/assets/68009/11475164/cf3c369e-9773-11e5-8165-d90329cd7197.png)

**Bonus problem**
We recently saw an issue with the image service where the Financial Times logo was being rendered as "Financial Tim". In the process of fixing the issue with the negative space in the A's I think we may have removed some of the problematic clipping areas/compound paths which were causing this problem.

AlbertoElias commented on svg/brand-ft-masthead.svg:
```diff
@@ -1 +1,10 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="1053.75" height="85" viewBox="0 0 1053.75 85"><defs><clipPath id="a"><path d="M0 67.96h842.2V0H0v67.96z"/></clipPath><clipPath id="b"><path d="M0 67.96h842.2V0H0v67.96z"/></clipPath></defs><path d="M25.24 72.65c0 5.97 1.63 7.33 12.62 7.33v3.67H0v-3.67c7.46 0 10.04-.68 10.04-7.2v-60.5c0-6.5-2.58-7.2-10.04-7.2V1.4h51.98c6.92 0 9.36-.26 11.53-.67l.5 23. ...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Can you csso this one?

AlbertoElias commented:
Doesn't the masthead look too bold?

Current one when properly loaded: 
![screen shot 2015-11-30 at 15 15 12](https://cloud.githubusercontent.com/assets/913117/11475402/311f9760-9775-11e5-85b8-a0f623e6300a.png)

alicebartlett commented:
OK, i've addressed all of the comments. The boldness of second logo is - I think - a poor rendering of the logo at that zoom level, in ie, in browserstack. It does look bold in that screen grab, but it looks no more or less bold than the one before it looked in ie at that zoom level.

@AlbertoElias happy to merge and then I'll do the patch release?
